search.go

```diff
@@ -74,12 +79,19 @@ type Appointments struct {
 	} `json:"centers"`
 }
 
-func timeNow() string {
-	return time.Now().Format("02-01-2006")
+func getTime() string {
+	if date == "" {
+		t := time.Now()
+		t = t.AddDate(0, 0, 1)
+		date = t.Format("02-01-2006")
+	}
+	log.Printf("Looking for appointment for date: %q", date)
+	return date
 }
 
 func queryServer(path string) ([]byte, error) {
 	req, err := http.NewRequest("GET", baseURL+path, nil)
+	log.Print(baseURL + path)
```

Comment:
Let's improve this message - something like :
```suggestion
	log.Print("Quering endpoint:", baseURL + path)
```
Also, is it possible to log it in debug mode?